src/Uno.Templates/content/unoapp/GitHub/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,4 @@
+blank_issues_enabled: false
+contact_links:
+  - name: "Learn More"
+    url: "https://platform.uno"
```

Comment:
Do we need to provide contact_links? I appreciate the marketing opportunity here but it's another thing developers have to change/remove. Let's remove this